tests/test_plugins_os_windows_lnk.py

```diff
@@ -0,0 +1,36 @@
+from flow.record.fieldtypes import datetime
+
+from dissect.target.plugins.os.windows.lnk import LnkPlugin, LnkRecord
+
+from ._utils import absolute_path
+
+
+def test_lnk(target_win, fs_win):
+    lnk_file = absolute_path("data/plugins/os/windows/lnk/pestudio.lnk")
+    fs_win.map_file("Users/pestudio.lnk", lnk_file)
+
+    target_win.add_plugin(LnkPlugin)
+
+    records = list(target_win.lnk(None))
+
+    assert len(records) == 1
+
+    assert isinstance(records[0], type(LnkRecord()))
+    assert str(records[0].lnk_path) == "sysvol\\users\\pestudio.lnk"
+    assert records[0].lnk_name is None
+    assert records[0].lnk_mtime == datetime("2023-04-12T08:50:02.008187+00:00")
```

Comment:
Which one is preferred, using the function `datetime` or converting the value to a string?

* `assert records[0].lnk_mtime == datetime("2023-04-12T08:50:02.008187+00:00")`
* `assert str(records[0].lnk_mtime) == "2023-04-12 08:50:02.008187+00:00"`